WordleGame/WordleGame/Views/Letter/LetterView.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  LetterView.swift
+//  WordleGame
+//
+//  Created by Alex on 3.05.22.
+//
+
+import Foundation
+import UIKit
+
+class LetterView: UIStackView {
+    
+    var letterBoxArray: [[LetterBox?]]
+    
+    init(letterBoxArray: [[LetterBox?]]) {
+        self.letterBoxArray = letterBoxArray
+        
+        super.init(frame: .zero)
+        
+        self.setUpView()
+    }
+    
+    required init(coder: NSCoder) {
+        self.letterBoxArray = []
+        
+        super.init(coder: coder)
+        
+        self.setUpView()
+    }
+    
+    private func setUpView() {
+        self.axis = .vertical
+        self.alignment = .center
+        self.distribution = .fillEqually
+        self.spacing = 4
+        
+        createLetterBox()
+    }
+    
+    private func createLetterBox() {
```

Comment:
I'm not able to see the game field. Please, check what can be the issue

![Simulator Screen Shot - iPhone 13 - 2022-05-11 at 22 46 02](https://user-images.githubusercontent.com/25828050/167933105-08db9326-ee88-4281-b1f3-521907c60af7.png)

Reply:
You can't see the game field because it is built from a random word that we get from the AllowedWords file. I get random word from this file, get count of letters and attempts. From this entry data I create game field. Sorry, it's my mistake. I created Resources folder and added AllowedWords.txt file to this folder. But I didn't add this file to folder outside the project. In this commit I added this file and you can check.